data/lib/compat/compat.lua

```diff
@@ -1312,3 +1312,65 @@ end
 function doPlayerTakeItem(cid, itemid, count)
 	return Player(cid):removeItem(itemid, count)
 end
+
+do
+	local combats = {
+		[COMBAT_PHYSICALDAMAGE] = 'physical',
+		[COMBAT_ENERGYDAMAGE] = 'energy',
+		[COMBAT_EARTHDAMAGE] = 'earth',
+		[COMBAT_FIREDAMAGE] = 'fire',
+		[COMBAT_UNDEFINEDDAMAGE] = 'undefined',
+		[COMBAT_LIFEDRAIN] = 'lifedrain',
+		[COMBAT_MANADRAIN] = 'manadrain',
+		[COMBAT_HEALING] = 'healing',
+		[COMBAT_DROWNDAMAGE] = 'drown',
+		[COMBAT_ICEDAMAGE] = 'ice',
+		[COMBAT_HOLYDAMAGE] = 'holy',
+		[COMBAT_DEATHDAMAGE] = 'death'
+	}
+
+	function getCombatName(combat)
+		return combats[combat]
+	end
+end
```

Comment:
Why wrapping with `do...end`?

Reply:
Just preference to enclose it with the function that's going to use the table. There's no need for that table to be used anywhere else.